core/src/main/java/com/novoda/landingstrip/animation/FastOutSlowInInterpolator.java

```diff
@@ -0,0 +1,45 @@
+package com.novoda.landingstrip.animation;
+
+class FastOutSlowInInterpolator extends LookupTableInterpolator {
```

Comment:
the `FastOutSlowInInterpolator` and `LookupTableInterpolator` are pulled directly from the design library